### PR TITLE
[NAE-1773] Action that sets null value to field does not trigger next actions

### PR DESCRIPTION
**Pull request by machacjozef**
# Description

Fixes [NAE-1773]

### Third party dependencies

No new dependencies were introduced

### Blocking Pull requests

There are no dependencies on other PR

## How Has Been This Tested?

Manually

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |   Linux Mint 20.3        |
| Runtime             |  Java 11        |
| Dependency Manager  |  Maven 3.8.4         |
| Framework version   |  Spring Boot 2.7.0     |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes have been checked, personally or remotely, with @SamuelPalaj      
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] My code follows the style guidelines of this project
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_application-engine)
    - [x] [Snyk](https://app.snyk.io/org/netgrif/project/ea82b3b8-658d-41f5-89a7-76cd600da01f)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1773]: https://netgrif.atlassian.net/browse/NAE-1773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/test/resources/insurance_portal_demo_test.xml [hunk 1]
```diff
@@ -608,6 +608,7 @@
         <values>3</values>
         <values>4</values>
         <values>viac ako 4</values>
+        <init>1</init>
```
Comment (minop) [inline src/test/resources/insurance_portal_demo_test.xml:611]: why was this added?

if this change was needed to pass some existing unit test, then it might hint at a bug caused by the change to the `EventService`

**Review by minop — APPROVED** [review]

**Review by mladoniczky — APPROVED** [review]
